### PR TITLE
Reverse color gradient

### PR DESCRIPTION
**Pull request by 6e656d6d61**

### CODE REVIEW

**Comment by zamarrowski** [comment]
Hi @6e656d6d61, thx for the PR but this option should be optional through react prop...

**Comment by 6e656d6d61** [comment]
That's right ! I'll change it.

**Comment by 6e656d6d61** [comment]
Done :smiley:

File: README.MD [hunk 1]
```diff
@@ -46,6 +46,7 @@ render(
 | `format` | `String` | Degrees format | `format="°C"` | '' |
 | `size` | `String` | Thermometer size. Can be small, normal or large | `size="large"` | normal |
 | `height` | `Number` | Thermometer height | `height={300}` | 200 |
+| `reverseGradient` | `boolean` | Reverse the gradient | `reverseGradient={"true"}` | false |
```
Comment (zamarrowski) [inline README.MD]: Now reverseGradiente expect a string not a boolean value. Change this to:

reverseGradient={true}

File: src/index.js [hunk 2]
```diff
@@ -42,6 +43,7 @@ class Thermometer extends Component {
       height: this.props.height || 200, //default 200
       valstr: () => this.options.value + this.options.format,
       percent: () => this.options.value / this.options.max * 100,
+      reverseGradient: this.props.reverseGradient === 'true' ? true : false, // default false
```
Comment (zamarrowski) [inline src/index.js]: If you change reverseGradient to boolean you can change this to: 

this.props.reverseGradient || false

File: README.MD (hunk 1 above)
Comment (6e656d6d61) [reply README.MD]: :+1:

File: src/index.js (hunk 2 above)
Comment (6e656d6d61) [reply src/index.js]: :+1:

**Pull request merged**